compose/config/config.py

```diff
@@ -478,7 +475,7 @@ def process_config_file(config_file, environment, service_name=None):
         'service',
         environment)
 
-    if config_file.version in (V2_0, V2_1, V3_0, V3_1):
+    if config_file.version > V1:
```

Comment:
That's a string comparison, right? Any chance this could break? Can we just say `if config_file.version != V1` instead?